### PR TITLE
Feature/organize modules

### PR DESCRIPTION
**Pull request by jmcarp**
The `__init__.py` file is approaching 500 lines and is getting increasingly disorganized. This patch splits up functionality into modules by file. This also wraps app invocation in a more standard `manage.py`, so the development app is started with `python manage.py runserver` rather than `python __init__.py`.

There's more cleanup that could be done here, but I think this is a reasonable start.

### CODE REVIEW

**Comment by ccostino** [comment]
@jmcarp:  If this is ready for review I'm happy to assign to myself since this covers a bunch of things we had spoken about one of the last times we paired/reviewed things with the repos.

**Comment by jmcarp** [comment]
@ccostino: sure, assigning to you.

**Comment by ccostino** [comment]
@jmcarp Thanks!  It looks like the previous merge I did caused some conflicts here, unfortunately.

File: openfecwebapp/__init__.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    return value if value not in nulls else None
+
+
+def get_election_url(candidate, cycle, district=None):
+    return url_for(
+        'elections',
+        office=candidate['office_full'].lower(),
+        state=nullify(candidate['state'], 'US'),
+        district=nullify(district or candidate['district'], '00'),
+        cycle=cycle,
```
Comment (ccostino) [inline openfecwebapp/__init__.py]: Do we need to update references of `cycle` to `election_year` given the filter changes in #1109 or is that specific to the templates only and taken care of just by that PR?

Comment (jmcarp) [inline openfecwebapp/__init__.py]: Not here--the election page doesn't accept an `election_year` query parameter.

File: openfecwebapp/__init__.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+from openfecwebapp.config import env
+from openfecwebapp import constants
+
+
+locale.setlocale(locale.LC_ALL, '')
+
+app = Flask(__name__, static_path='/static', static_folder='../dist')
+
+from openfecwebapp import routes  # noqa
+from openfecwebapp import filters  # noqa
```
Comment (jmcarp) [inline openfecwebapp/__init__.py]: This ignores a flake8 warning for late imports.

File: openfecwebapp/__init__.py [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+def _get_default_cycles():
+    cycle = utils.current_cycle()
+    return list(range(cycle - 4, cycle + 2, 2))
+
+
+def series_has_data(values, key):
+    return next(
+        (True for value in values if value.get(key) is not None),
+        False,
+    )
```
Comment (jmcarp) [inline openfecwebapp/__init__.py]: Honestly, I don't remember--this is all old code that would ideally be torn out and replaced. Feel free @ccostino!

**Comment by jmcarp** [comment]
Responded to comments.

File: openfecwebapp/__init__.py (hunk 2 above)
Comment (ccostino) [inline openfecwebapp/__init__.py]: :+1:

File: openfecwebapp/__init__.py (hunk 3 above)
Comment (ccostino) [inline openfecwebapp/__init__.py]: Hahaha, fair enough!  Adding to my list of things to investigate further in the future. :-)

File: openfecwebapp/__init__.py (hunk 1 above)
Comment (ccostino) [inline openfecwebapp/__init__.py]: Okay, I just wasn't sure.  So only the candidate page received the change to accept the `election_year` parameter, correct?

**Comment by ccostino** [comment]
Just one final question and a sync up with the `develop` branch and I think this is good to go as well after reviewing.

**Comment by jmcarp** [comment]
I'm wondering if github is acting up here--I don't see conflicts between this branch and develop on the command line.

File: openfecwebapp/__init__.py (hunk 1 above)
Comment (jmcarp) [inline openfecwebapp/__init__.py]: "Candidate page" is ambiguous--we have a candidate detail page that shows one candidate (/candidate/P012345/), a generic candidate browse page that shows all candidates (/candidates/), and office-specific candidate browse pages that show all candidates by office (/candidates/senate/). The changes in #1109 targeted the generic candidate browse page.

Comment (ccostino) [inline openfecwebapp/__init__.py]: Ah okay, thanks for clarifying!

**Comment by ccostino** [comment]
You're right, I just double checked this myself and while there are some changes that could be merged in, nothing causes a conflict.

**Comment by ccostino** [comment]
After syncing up in our weekly meeting, I am just going to merge this on the command line and will close this PR after done.

**Pull request merged**